app/code/Magento/Checkout/view/frontend/web/js/model/quote.js

```diff
@@ -57,13 +57,6 @@ define([
         },
 
         /**
-         * @return {*}
-         */
-        getItems: function () {
```

Comment:
According to the [Backwards Compatible Development Guide](http://devdocs.magento.com/guides/v2.1/ext-best-practices/extension-coding/backwards-compatible-development/index.html#js) this method can not be removed, since it may break some existing extension relying on it or the data it provides.